README.md

```diff
@@ -3,51 +3,50 @@
 Inertia makes it easy to set up automated deployment for Dockerized
 applications.
 
+## Installation
+
 ```bash
 go get -u github.com/ubclauncpad/inertia
 ```
 
-## Deploy an Application
+Alternatively, you can download Inertia from the [Releases](https://github.com/ubclaunchpad/inertia/releases) page.
+
+## Usage
 
-Applications are deployed over SSH. You will need an SSH username and PEM file
-to get started. Inside of a git repository, run the following:
+Inside of a git repository, run the following:
 
 ```bash
 $> inertia init
 
 $> inertia remote add glcoud 35.227.171.49 -u root -i /path/to/my/.ssh/id_rsa
 Remote 'glcoud' added.
 
-$> inertia remote bootstrap gcloud
-Bootstrapping remote...
+$> inertia gcloud init
```

Comment:
`inertia [REMOTE] init`

Reply:
My bad, I know.

Reply:
I think using `gcloud` here is good since it's a specific example (line before is `inertia remote add gcloud`)